content/en/docs/spinnaker/terraform-enable-integration.md

```diff
@@ -268,53 +257,12 @@ spec:
       armory:
         terraform:
           enabled: true
-          git:
-            enabled: true
-            username: my-user # BitBucket user name.
-            accessToken: abc  # BitBucket password. This field supports "encrypted" field references (https://docs.armory.io/spinnaker-install-admin-guides/secrets/)
-```
-
-**Halyard**
-
-```
-# This will prompt for the token, which is your BitBucket password
-hal armory terraform edit \
-  --git-enabled \
-  --git-username <USERNAME> \
-  --git-access-token
-```
-
-Next, go to [Enabling the Terraform UI](#enabling-the-terraform-integration-ui) if you are using an Armory Spinnaker version before 2.17.3. Otherwise, go to [Completing the installation](#completing-the-installation).
-
-## Enabling the Terraform Integration UI
-
-If you previously used the Terraform Integration stage by editing the JSON representation of the stage, those stages are automatically converted to use the UI.
-
-For Armory Spinnaker 2.17.3 and later, the UI is on by default. If you are using an earlier version, manually enable the stage UI for Deck:
-
-**Operator**
-
-Edit the `SpinnakerService` manifest to add the following:
-
-```yaml
-apiVersion: spinnaker.armory.io/{{< param operator-extended-crd-version >}}
-kind: SpinnakerService
-metadata:
-  name: spinnaker
-spec:
-  spinnakerConfig:
-    profiles:
-      deck:
-        settings-local.js: |
-          window.spinnakerSettings.feature.terraform = true;
 ```
 
 **Halyard**
 
-Edit `~/.hal/default/profiles/settings-local.js` and add the following:
-
 ```
-window.spinnakerSettings.feature.terraform = true;
```

Comment:
I'm not sure we can remove this yet. We haven't removed the feature flag or made Halyard add the flag by itself.

Reply:
Since the docs are versioned now, this removal would affect 2.20 and higher docs, which have the UI on by default. The config would still be there for lower versions of the doc.

Reply:
Apparently we were lying when we said it was on by default in 2.17.3. I'm not sure how that happened. I checked the code and the [feature flag is still in there](https://github.com/armory-io/deck-armory/blob/140e9efe6485b547921f216c7b3be97960848330/src/spinnaker.modules/terraform/stage/terraformStage.ts#L15)

Reply:
So I'm adding this block back in and removing the line about it being on by default